compiler/rustc_codegen_llvm/src/back/write.rs

```diff
@@ -499,7 +498,7 @@ pub(crate) unsafe fn optimize_with_new_llvm_pass_manager(
 }
 
 // Unsafe due to LLVM calls.
-pub(crate) unsafe fn optimize(
+pub(crate) unsafe fn maybe_optimize(
```

Comment:
I'd prefer to keep the name here (because this implements the public `optimize` API) and instead call the other one `optimize_with_pass_manager` or something.

Reply:
Ok, I don't want to bikeshed too much, but how about `llvm_optimize` for the inner part?

Reply:
Sounds good

Reply:
Done!